src/main/webapp/resources/js/apis/files/sequence-files.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * FastQC related API functions
+ */
+
+import { setBaseUrl } from "../../utilities/url-utilities";
+import axios from "axios";
+
+const BASE_URL = setBaseUrl(`/ajax/sequenceFiles`);
+
+/*
+ * Get the fastqc details.
+ * @param {number} sequencingObjectId SequencingObject ID
+ * @param {number} sequenceFileId SequenceFile ID
+ * @return {Promise<*>} `data` contains the OK response and a dto with the fastqc details;
+ *                      `error` contains error information if an error occurred.
+ */
+export async function getFastQCDetails(sequencingObjectId, sequenceFileId) {
+  try {
+    const { data } = await axios.get(`${BASE_URL}/fastqc-details`, {
+      params: {
+        sequencingObjectId,
+        sequenceFileId
+      }
+    });
+    return data;
+  }  catch (error) {
+    return { error };
```

Comment:
Aren't we throwing errors.

Reply:
Updated in 8d915d2b5135ef05b2f4eafaebe586d755b13903